components/CodeBlock/CodeBlock.tsx

```diff
@@ -15,11 +16,15 @@ interface ICodeBlockProps {
 
 export const CodeBlock = ({ codeSnippet, languageType }: ICodeBlockProps) => {
 	return (
-		<SyntaxHighlighter
-			language={languageType}
-			style={dracula}
-			wrapLongLines={true}>
-			{codeSnippet}
-		</SyntaxHighlighter>
+		<div style={{ position: "relative" }}>
```

Comment:
Please move all inline styles to a module.css file. If one doesn't yet exist for a component, feel free to make one.

Reply:
Yep, of course. Have jjust made a new CSS module in the directory